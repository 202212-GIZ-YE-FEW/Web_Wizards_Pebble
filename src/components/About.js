const About = () => {
    return (
        <section id='about'>
            <div
                className='
                    relative 
                    overflow-hidden 
                    bg-no-repeat 
                    bg-cover 
                    w-full
                    brightness-200
                    h-[231px]
                    md:h-[493px]
                    lg:h-[690px]
                    xl:h-[787px]
                    overlay-container
                '
                style={{
                    backgroundPosition: "center",
                    backgroundImage: "url(/images/about.png)",
                }}
            ></div>
            <div
                className='
                    absolute
                    inset-0
                    opacity-60
                    text-center
                    lg:relative
                '
            >
                <h2
                    className='
                        font-black
                        top-1/2
                        text-xl
                        md:text-4xl
                        lg:text-5xl
                        text-dark
                        mt-16
                        mb-4
                        md:mt-52
                    '
                >
                    About Pebble Works
                </h2>
                <p
                    className='
                        text-sm
                        relative
                        mx-auto
                        w-1/2
                        md:text-xl
                        lg:text-3xl
                    '
                >
                    Body placeholder for text paragraph. A paragraph is a
                    self-contained unit of text dealing with a particular point
                    or idea.
                </p>
            </div>
        </section>
    );
};

export default About;
