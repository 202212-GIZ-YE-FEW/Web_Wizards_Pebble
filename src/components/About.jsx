function Brief(props) {
    const { head, brief } = props;
    return (
        <div className='absolute inset-0 mx-auto translate-y-1/4 h-full w-full flex-col items-center justify-center lg:relative'>
            <h2
                className='
                m-1 text-center text-xl mdP:text-3xl font-bold md:m-4 md:text-5xl md:font-bold lg:text-5xl lg:font-bold'
            >
                {head}
            </h2>
            <p
                className='
                text-sm mr-12 ml-12 text-center md:ml-24 md:mr-24 md:text-2xl lg:ml-32 lg:mr-32 lg:text-2xl'
            >
                {brief}
            </p>
        </div>
    );
}

const About = (props) => {
    const { t } = props;

    return (
        <section id='about'>
            <div
                className='relative overflow-hidden  bg-no-repeat  bg-cover w-full top-[-40px] brightness-200 h-[214px] sm:h-[489px] lg:h-[689px] xl:h-[789px] 2xl:h-[889px] before:absolute before:inset-0 before:bg-zinc-50 before:opacity-10'
                style={{
                    backgroundPosition: "center",
                    backgroundImage: "url(/images/about.png)",
                }}
            ></div>
            <Brief head={t("about")} brief={t("brief")} />
        </section>
    );
};

export default About;
