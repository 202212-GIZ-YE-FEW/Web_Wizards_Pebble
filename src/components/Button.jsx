import React from "react";

function Button({
    classes,
    children,
    handleClick = () => {},
    ariaLabel,
    dropDownButton = null,
}) {
    // console.log(classes);
    return (
        <>
            {dropDownButton === null ? (
                <button
                    onClick={() => handleClick(children)}
                    className={`button ${classes}`}
                >
                    {children}
                </button>
            ) : (
                <button
                    onClick={() => handleClick(children)}
                    className={`button ${classes}`}
                    aria-label={ariaLabel}
                    data-te-offcanvas-toggle
                    data-te-target={dropDownButton.target}
                    aria-controls={dropDownButton.controls}
                    data-te-ripple-init
                    data-te-ripple-color='dark'
                >
                    {children}
                </button>
            )}
        </>
    );
}

export default Button;
