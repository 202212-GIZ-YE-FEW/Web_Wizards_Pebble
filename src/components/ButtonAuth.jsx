import Image from "next/image";
import { Button } from "react-flatifycss";

import "flatifycss/dist/css/flatify-min.css";
import styles from "./ButtonAuth.module.css";

const ButtonAuth = (props) => {
    const { icon, alt, width, height } = props
    return (
        <>
            <Button
                className={`w-full h-14 px-6 transition ${styles.ButtonAuth}`}
            >
                <div className='relative flex items-center space-x-4 justify-center'>
                    <Image
                        src={icon}
                        alt={alt}
                        width={width}
                        height={height}
                    />
                    <span className='block w-max font-medium tracking-wide text-gray-900 sm:text-base'>
                        {props.text}
                    </span>
                </div>
            </Button>
        </>
    );
};

export default ButtonAuth;
