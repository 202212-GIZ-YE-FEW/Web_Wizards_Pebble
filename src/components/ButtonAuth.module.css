.ButtonAuth {
    background-color: white;
    border: 2px solid black;
    border-radius: 0.5rem;
    box-shadow: 2px 2px #1a1a1a;
}
