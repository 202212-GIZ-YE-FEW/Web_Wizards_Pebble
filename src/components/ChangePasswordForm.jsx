import EditprofileButton from "./EditProfileButton";
import InputForm from "./InputForm";
import SaveButton from "./SaveButton";
import SecondaryTitle from "./SecondaryTitle";
const ChangePasswordForm = ({ t }) => {
    return (
        <div className='w-11/12 mx-auto rounded-3xl flex flex-col justify-center bg-[#d4f3fa] md:px-10 px-2 py-10'>
            <SecondaryTitle text={t("changePassword")} />
            <div className='flex flex-row py-6'>
                <InputForm
                    placeholder={t("newPassword")}
                    label={t("newPassword")}
                    name='New Password'
                    type='Password'
                    hasFloatingLabel
                    togglePasswordLabel='Show/Hide password'
                    togglePassword='true'
                />
                <InputForm
                    placeholder={t("confirmPassword")}
                    label={t("confirmPassword")}
                    name='Confirm Password'
                    type='Password'
                    hasFloatingLabel
                    togglePasswordLabel='Show/Hide password'
                    togglePassword='true'
                />
            </div>

            <div className='flex flex-row '>
                <EditprofileButton text={t("cancel")} />
                <SaveButton text={t("submit")} />
            </div>
        </div>
    );
};

export default ChangePasswordForm;
