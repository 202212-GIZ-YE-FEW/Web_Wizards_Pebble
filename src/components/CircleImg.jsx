const CircleImg = () => {
    return (
        <div className='w-2/12 '>
            <div className='flex items-center justify-center w-32 h-32 md:w-36 md:h-36 lg:w-64 lg:h-64 rounded-full bg-black-100 text-white text-3xl font-bold'>
                <span className=''>R</span>
            </div>
        </div>
    );
};

export default CircleImg;
