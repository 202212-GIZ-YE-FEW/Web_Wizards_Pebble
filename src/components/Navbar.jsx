import Link from "next/link";
import { usePathname } from "next/navigation";
import { withTranslation } from "next-i18next";
import {
    Button,
    Dropdown,
    DropdownBody,
    DropdownButton,
} from "react-flatifycss";

import Logo from "@/components/Logo";

//TODO:  do dyanmic routers
//
// read paths of pages dir and extract their respective name and href

const routes = [
    { name: "Events", href: "/events" },
    { name: "About", href: "/about" },
];

function Nav(props) {
    const { name, t, href } = props;
    return <Link href={href}>{t(name)}</Link>;
}

function SwitchLangDropDown(props) {
    const { to, lang } = props;
    return (
        <Dropdown autoClose offsetX={-40} id='lang-dropdown'>
            <DropdownButton buttonStyle={false} style={{ color: "white" }}>
                {lang.toUpperCase()}
            </DropdownButton>
            <DropdownBody isMenu className='rounded-none'>
                {
                    //TODO: do dynamic languages
                }
                <li className='menu-item'>
                    <Link
                        href={to}
                        locale='en'
                        onClick={() => (document.dir = "ltr")}
                    >
                        English
                    </Link>
                </li>
                <li className='menu-item'>
                    <Link
                        href={to}
                        locale='ar'
                        onClick={() => (document.dir = "rtl")}
                    >
                        العربية
                    </Link>
                </li>
            </DropdownBody>
        </Dropdown>
    );
}

function MobileDropDown(props) {
    const { t } = props;
    return (
        <Dropdown
            autoClose
            id='lang-dropdown'
            className='md:hidden place-self-center'
            offsetX={65}
        >
            <DropdownButton buttonStyle={false} hasArrow={false}>
                <svg
                    width='25'
                    height='19'
                    viewBox='0 0 25 19'
                    fill='none'
                    xmlns='http://www.w3.org/2000/svg'
                >
                    <path
                        fillRule='evenodd'
                        clipRule='evenodd'
                        d='M22.955 0.664062H1.67799C-0.348392 0.664062 -0.348392 2.69044 1.67799 2.69044H22.955C24.9813 2.69044 24.9813 0.664062 22.955 0.664062ZM1.67799 8.76958H22.955C24.9813 8.76958 24.9813 10.796 22.955 10.796H1.67799C-0.348392 10.796 -0.348392 8.76958 1.67799 8.76958ZM1.67799 16.8751H22.955C24.9813 16.8751 24.9813 18.9015 22.955 18.9015H1.67799C-0.348392 18.9015 -0.348392 16.8751 1.67799 16.8751Z'
                        fill='white'
                    />
                </svg>
            </DropdownButton>
            <DropdownBody
                isMenu
                className='shadow-[2px_2px_0px_#1A1A1A;] border-2 border-[#1A1A1A] rounded-none'
            >
                {
                    //TODO: do dynamic languages
                }
                {routes.map((route, index) => {
                    return (
                        <li className='menu-item' key={index}>
                            <Nav t={t} name={route.name} href={route.href} />
                        </li>
                    );
                })}
            </DropdownBody>
        </Dropdown>
    );
}

export function Navbar(props) {
    const path = usePathname() || "/";
    // const query = useSearchParams();
    const { t, i18n } = props;
    const lang = i18n.language.toUpperCase();

    return (
        <nav className='bg-primary-200 h-[120px] max-w-full flex items-center mb-10'>
            <div className='container flex md:justify-between'>
                <MobileDropDown t={t} />
                <Link
                    href='/'
                    className='mr-1 justify-center flex flex-1 md:flex-none'
                >
                    <Logo />
                </Link>
                <div className='hidden md:flex gap-[10px]'>
                    <Link href='/signin'>
                        {" "}
                        <Button className='w-[113px] h-[52px] !rounded-[8px] !py-[11px] !px-[16px] !bg-white !shadow-[2px_2px_0px_#1A1A1A;] !border-2 !border-[#1A1A1A] !border-solid !text-black-100'>
                            {t("signIn")}
                        </Button>
                    </Link>
                    <Link href='/signup'>
                        <Button className='w-[113px] h-[52px] !rounded-[8px] !py-[11px] !px-[16px] !bg-[#2F7DA9] !text-white !shadow-none !border-none'>
                            {t("signUp")}
                        </Button>
                    </Link>
                    <div className='flex items-center gap-1'>
                        <svg
                            width='28'
                            height='28'
                            viewBox='0 0 28 28'
                            fill='none'
                            xmlns='http://www.w3.org/2000/svg'
                        >
                            <path
                                fillRule='evenodd'
                                clipRule='evenodd'
                                d='M14 27.75C21.5939 27.75 27.75 21.5939 27.75 14C27.75 6.40608 21.5939 0.25 14 0.25C6.40608 0.25 0.25 6.40608 0.25 14C0.25 21.5939 6.40608 27.75 14 27.75ZM11.6258 25.2121C9.49226 21.8802 8.30818 18.5222 8.08965 15.1458H2.59824C3.09437 20.1428 6.80075 24.1954 11.6258 25.2121ZM8.08965 12.8542C8.30818 9.47784 9.49226 6.11978 11.6258 2.78795C6.80075 3.80459 3.09437 7.85719 2.59824 12.8542H8.08965ZM19.9104 12.8542C19.6918 9.47784 18.5077 6.11978 16.3742 2.78795C21.1993 3.80459 24.9056 7.85719 25.4018 12.8542H19.9104ZM16.3742 25.2121C18.5077 21.8802 19.6918 18.5222 19.9104 15.1458H25.4018C24.9056 20.1428 21.1993 24.1954 16.3742 25.2121ZM10.3869 12.8542C10.621 9.71035 11.8191 6.54607 14 3.35196C16.1809 6.54607 17.379 9.71035 17.6131 12.8542H10.3869ZM14 24.648C11.8191 21.4539 10.621 18.2896 10.3869 15.1458H17.6131C17.379 18.2896 16.1809 21.4539 14 24.648Z'
                                fill='white'
                            />
                        </svg>
                        <SwitchLangDropDown to={path} lang={lang} />
                    </div>
                </div>
            </div>
        </nav>
    );
}

export default withTranslation("common")(Navbar);
