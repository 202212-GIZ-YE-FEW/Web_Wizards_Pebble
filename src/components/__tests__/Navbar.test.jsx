import renderer from "react-test-renderer";

import { Navbar } from "@/components/Navbar";

it("renders correctly", () => {
    const tree = renderer
        .create(<Navbar t={(str) => str} i18n={{ language: "en" }} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
