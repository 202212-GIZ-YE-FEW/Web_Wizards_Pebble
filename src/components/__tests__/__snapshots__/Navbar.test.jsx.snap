// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="bg-primary-200 h-[120px] mb-10"
>
  <div
    className="container flex h-full md:justify-between items-center"
  >
    <div
      className="dropdown__DropdownWrapper-sc-clud83-0 cXRnlz dropdown-wrapper md:hidden place-self-center md:hidden place-self-center"
      id="wrapper-mobile-dropdown"
    >
      <button
        aria-expanded={false}
        className="button__ButtonWrapper-sc-1aw98b4-0 button dropdown-button__DropdownButtonWrapper-sc-188aygt-0 dropdown-toggle arrow-bottom no-style"
        disabled={false}
        id="e088a42c-ef7a-5d"
        onClick={[Function]}
      >
        <svg
          fill="none"
          height="19"
          viewBox="0 0 25 19"
          width="25"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clipRule="evenodd"
            d="M22.955 0.664062H1.67799C-0.348392 0.664062 -0.348392 2.69044 1.67799 2.69044H22.955C24.9813 2.69044 24.9813 0.664062 22.955 0.664062ZM1.67799 8.76958H22.955C24.9813 8.76958 24.9813 10.796 22.955 10.796H1.67799C-0.348392 10.796 -0.348392 8.76958 1.67799 8.76958ZM1.67799 16.8751H22.955C24.9813 16.8751 24.9813 18.9015 22.955 18.9015H1.67799C-0.348392 18.9015 -0.348392 16.8751 1.67799 16.8751Z"
            fill="white"
            fillRule="evenodd"
          />
        </svg>
      </button>
      <ul
        aria-labelledby="e088a42c-ef7a-5d"
        className="dropdown-body__DropdownBodyWrapper-sc-xo3k2p-0 dropdown menu-items-wrapper shadow-[2px_2px_0px_#1A1A1A] border-2 border-[#1A1A1A] rounded-none"
        onClick={[Function]}
        style={
          Object {
            "left": "0",
            "position": "absolute",
            "top": "0",
          }
        }
      >
        <li
          className="menu-item hover:!bg-primary-100 !rounded-none"
        >
          <a
            className=""
            href="/events"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            allEvents
          </a>
        </li>
        <li
          className="menu-item hover:!bg-primary-100 !rounded-none"
        >
          <a
            className=""
            href="/about"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            about
          </a>
        </li>
        <li
          className="menu-item hover:!bg-primary-100 !rounded-none"
        >
          <a
            className=""
            href="/#how_it_works"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            howItWorks
          </a>
        </li>
        <li
          className="menu-item hover:!bg-primary-100 !rounded-none"
        >
          <a
            className=""
            href="/signin"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            signIn
          </a>
        </li>
        <li
          className="menu-item hover:!bg-primary-100 !rounded-none"
        >
          <a
            className=""
            href="/signup"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            signUp
          </a>
        </li>
        <div
          className="dropdown__DropdownWrapper-sc-clud83-0 cXRnlz dropdown-wrapper"
          id="wrapper-lang-dropdown"
        >
          <button
            aria-expanded={false}
            className="button__ButtonWrapper-sc-1aw98b4-0 button dropdown-button__DropdownButtonWrapper-sc-188aygt-0 dropdown-toggle items-center gap-1 menu-item hover:!bg-primary-100 !flex !px-4 !py-3 !rounded-none !w-full arrow-bottom no-style"
            disabled={false}
            id="47b38185-4a78-57"
            onClick={[Function]}
          >
            <svg
              fill="none"
              height="28"
              viewBox="0 0 28 28"
              width="28"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clipRule="evenodd"
                d="M14 27.75C21.5939 27.75 27.75 21.5939 27.75 14C27.75 6.40608 21.5939 0.25 14 0.25C6.40608 0.25 0.25 6.40608 0.25 14C0.25 21.5939 6.40608 27.75 14 27.75ZM11.6258 25.2121C9.49226 21.8802 8.30818 18.5222 8.08965 15.1458H2.59824C3.09437 20.1428 6.80075 24.1954 11.6258 25.2121ZM8.08965 12.8542C8.30818 9.47784 9.49226 6.11978 11.6258 2.78795C6.80075 3.80459 3.09437 7.85719 2.59824 12.8542H8.08965ZM19.9104 12.8542C19.6918 9.47784 18.5077 6.11978 16.3742 2.78795C21.1993 3.80459 24.9056 7.85719 25.4018 12.8542H19.9104ZM16.3742 25.2121C18.5077 21.8802 19.6918 18.5222 19.9104 15.1458H25.4018C24.9056 20.1428 21.1993 24.1954 16.3742 25.2121ZM10.3869 12.8542C10.621 9.71035 11.8191 6.54607 14 3.35196C16.1809 6.54607 17.379 9.71035 17.6131 12.8542H10.3869ZM14 24.648C11.8191 21.4539 10.621 18.2896 10.3869 15.1458H17.6131C17.379 18.2896 16.1809 21.4539 14 24.648Z"
                fill="#2e4051"
                fillRule="evenodd"
              />
            </svg>
            EN
          </button>
          <ul
            aria-labelledby="47b38185-4a78-57"
            className="dropdown-body__DropdownBodyWrapper-sc-xo3k2p-0 dropdown menu-items-wrapper shadow-[2px_2px_0px_#1A1A1A] border-2 border-[#1A1A1A] rounded-none"
            onClick={[Function]}
            style={
              Object {
                "left": "0",
                "position": "absolute",
                "top": "0",
              }
            }
          >
            <li
              className="menu-item"
            >
              <a
                href="/"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
              >
                English
              </a>
            </li>
            <li
              className="menu-item"
            >
              <a
                href="/"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
              >
                العربية
              </a>
            </li>
            <li
              aria-hidden={false}
            >
              <span
                className="pointer-arrow"
                style={
                  Object {
                    "position": "absolute",
                  }
                }
              />
            </li>
          </ul>
        </div>
        <li
          aria-hidden={false}
        >
          <span
            className="pointer-arrow"
            style={
              Object {
                "position": "absolute",
              }
            }
          />
        </li>
      </ul>
    </div>
    <a
      className="mr-1 justify-center flex flex-1 md:flex-none"
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      <img
        alt="PebbleWork Logo"
        className="max-h-[32.32px] max-w-[100.75px] md:max-h-full md:max-w-full"
        data-nimg="1"
        decoding="async"
        height={69.13}
        onError={[Function]}
        onLoad={[Function]}
        src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
        srcSet="/_next/image?url=%2Fimg.jpg&w=256&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
        style={
          Object {
            "color": "transparent",
          }
        }
        width={215.44}
      />
    </a>
    <div
      className="flex gap-[10px] !items-center"
    >
      <a
        className="hover:!no-underline"
        href="/signin"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
        role="button"
      >
        <button
          className="button__ButtonWrapper-sc-1aw98b4-0 button hidden md:block w-[113px] h-[52px] !rounded-[8px] !py-[11px] !px-[16px] !bg-white !shadow-[2px_2px_0px_#1A1A1A] !border-2 !border-[#1A1A1A] !border-solid !text-black-100"
          disabled={false}
        >
          signIn
        </button>
      </a>
      <a
        className="hover:!no-underline"
        href="/signup"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
        role="button"
      >
        <button
          className="button__ButtonWrapper-sc-1aw98b4-0 button hidden md:block w-[113px] h-[52px] !rounded-[8px] !py-[11px] !px-[16px] !bg-[#2F7DA9] !text-white !shadow-none !border-none"
          disabled={false}
        >
          signUp
        </button>
      </a>
      <div
        className="dropdown__DropdownWrapper-sc-clud83-0 cXRnlz dropdown-wrapper"
        id="wrapper-lang-dropdown"
      >
        <button
          aria-expanded={false}
          className="button__ButtonWrapper-sc-1aw98b4-0 button dropdown-button__DropdownButtonWrapper-sc-188aygt-0 dropdown-toggle items-center gap-1 hidden md:flex !text-white arrow-bottom no-style"
          disabled={false}
          id="47b38185-4a78-57"
          onClick={[Function]}
        >
          <svg
            fill="none"
            height="28"
            viewBox="0 0 28 28"
            width="28"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clipRule="evenodd"
              d="M14 27.75C21.5939 27.75 27.75 21.5939 27.75 14C27.75 6.40608 21.5939 0.25 14 0.25C6.40608 0.25 0.25 6.40608 0.25 14C0.25 21.5939 6.40608 27.75 14 27.75ZM11.6258 25.2121C9.49226 21.8802 8.30818 18.5222 8.08965 15.1458H2.59824C3.09437 20.1428 6.80075 24.1954 11.6258 25.2121ZM8.08965 12.8542C8.30818 9.47784 9.49226 6.11978 11.6258 2.78795C6.80075 3.80459 3.09437 7.85719 2.59824 12.8542H8.08965ZM19.9104 12.8542C19.6918 9.47784 18.5077 6.11978 16.3742 2.78795C21.1993 3.80459 24.9056 7.85719 25.4018 12.8542H19.9104ZM16.3742 25.2121C18.5077 21.8802 19.6918 18.5222 19.9104 15.1458H25.4018C24.9056 20.1428 21.1993 24.1954 16.3742 25.2121ZM10.3869 12.8542C10.621 9.71035 11.8191 6.54607 14 3.35196C16.1809 6.54607 17.379 9.71035 17.6131 12.8542H10.3869ZM14 24.648C11.8191 21.4539 10.621 18.2896 10.3869 15.1458H17.6131C17.379 18.2896 16.1809 21.4539 14 24.648Z"
              fill="white"
              fillRule="evenodd"
            />
          </svg>
          EN
        </button>
        <ul
          aria-labelledby="47b38185-4a78-57"
          className="dropdown-body__DropdownBodyWrapper-sc-xo3k2p-0 dropdown menu-items-wrapper shadow-[2px_2px_0px_#1A1A1A] border-2 border-[#1A1A1A] rounded-none"
          onClick={[Function]}
          style={
            Object {
              "left": "0",
              "position": "absolute",
              "top": "0",
            }
          }
        >
          <li
            className="menu-item"
          >
            <a
              href="/"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              English
            </a>
          </li>
          <li
            className="menu-item"
          >
            <a
              href="/"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              العربية
            </a>
          </li>
          <li
            aria-hidden={false}
          >
            <span
              className="pointer-arrow"
              style={
                Object {
                  "position": "absolute",
                }
              }
            />
          </li>
        </ul>
      </div>
    </div>
  </div>
</nav>
`;
